src/resources/account/account.helper.js

```diff
@@ -0,0 +1,38 @@
+const psl = require('psl');
+const url = require('url');
+const { createTokens, removeTokens } = require('resources/token/token.helper');
+const { webUrl } = require('config');
+const { COOKIES } = require('app.constants');
+
+exports.setTokens = async (ctx, userId) => {
+  const { accessTokenEntity, refreshTokenEntity } = await createTokens(userId);
+
+  const parsedUrl = url.parse(webUrl);
+  const parsed = psl.parse(parsedUrl.hostname);
+  const cookiesDomain = parsed.domain;
+
+  ctx.cookies.set(COOKIES.ACCESS_TOKEN, accessTokenEntity.value, {
+    httpOnly: false,
+    expires: accessTokenEntity.expireAt,
+    domain: cookiesDomain,
+  });
+
+  ctx.cookies.set(COOKIES.REFRESH_TOKEN, refreshTokenEntity.value, {
+    httpOnly: true,
+    expires: refreshTokenEntity.expireAt,
+    domain: cookiesDomain,
+  });
+
+  ctx.cookies.set(COOKIES.EXPIRED_AT, accessTokenEntity.expireAt, {
```

Comment:
Do we need a separate cookie for expire_at? It's already stored in access token cookie